contracts/core/Ownable.sol

```diff
@@ -17,4 +17,7 @@ contract Ownable is OwnableUpgradeable {
       _transferOwnership(owner);
     }
   }
+
+  // add padding as storage layout changed in OZ contracts v4
+  uint256[1] private ____gap_Ownable;
```

Comment:
is it possible to actually use these with the latest hardhat upgrades lib from OZ?

Reply:
I have figured this out now, to enable it I had to:

1. Use `prepareUpgrade` to get the new storage layout metadata output in the json files
2. As part of the upgrade, fix that metadata to point to the newly deployed contract
3. Future upgrades work as normal.

This is tested by:

1. Start a local forked node `yarn start-forked-node:sokol`
2. Run the migration against the fork: `yarn migrate:fork:sokol`
3. Make a change to an arbitrary contract
4. Deploy to the forked node: `yarn deploy:fork:sokol`


Checked that the checks are still good by making a bad change:
![image](https://user-images.githubusercontent.com/93225030/152440014-a9a72571-a254-4618-bf95-c1db04c83ea8.png)

Made a good change and deployed:

![image](https://user-images.githubusercontent.com/93225030/152446176-45cc70c4-af00-4a96-a0dc-0a318ac3cee4.png)


Verified working storage layout changes after deploy:
![image](https://user-images.githubusercontent.com/93225030/152446395-d39c46ef-1699-4804-ada7-49d1ba9d44b7.png)

Reply:
that is SOOOO nice!